podcasts/NowPlayingPlayerItemViewController.swift

```diff
@@ -119,6 +119,8 @@ class NowPlayingPlayerItemViewController: PlayerItemViewController {
 
     @IBOutlet var timeSlider: TimeSlider! {
         didSet {
+            timeSlider.accessibilityLabel = L10n.accessibilityEpisodePlayback
+            timeSlider.accessibilityHint = L10n.accessibilityEpisodePlaybackHint
```

Comment:
I don't think we need to manually set this hint. On my tests the "Swipe up or down with one finger to adjust the value" is read twice because it's already added by the system.